### PR TITLE
[4.0] [plg_system_httpheaders] Drop option "Write headers to the configuration file"

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #25716

### Summary of Changes

As discussed here #25716 this here is removing the htaccess / web.config option from the plugin again for the reasons explained in that issue.

### Testing Instructions

Make sure the option is gone and nothing writes to the htaccess and web.config files.
make sure headers are still set correctly.

### Expected result

nothing writes to the htaccess and web.config files.

### Actual result

this is a option to write to the htaccess and web.config files.

### Documentation Changes Required

Yes.

### CODE REVIEW

File: plugins/system/httpheaders/httpheaders.php [hunk 1]
```diff
@@ -830,14 +408,14 @@ private function getStaticHeaderConfiguration($pluginParams = false): array
 	 */
 	private function setStaticHeaders(): void
 	{
-		$this->staticHeaderConfiguration = $this->getStaticHeaderConfiguration($this->params);
+		$staticHeaderConfiguration = $this->getStaticHeaderConfiguration($this->params);
```
Comment (Quy) [inline plugins/system/httpheaders/httpheaders.php]:
```suggestion
		$staticHeaderConfiguration = $this->getStaticHeaderConfiguration();
```

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 82630e37d28450dee7c5b08d60ff77c82bd0a036<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25754">issues.joomla.org/tracker/joomla-cms/25754</a>.</sub>

**Comment by zero-24** [comment]
Thanks for your test @SharkyKZ 👍

**Comment by zero-24** [comment]
As this here in the end removes an feature from the stable build i would like to see it merged before we hit beta.

cc @wilsonge

**Comment by wilsonge** [comment]
Thanks! Frustrating it didn't work out

**Comment by zero-24** [comment]
The option has not been documented in the fist place so it does not need to be removed :D

**Pull request merged**